app/views/welcome/support_us.html.erb

```diff
@@ -76,4 +78,4 @@
 		<p>
 			We wouldn't be able to do what we do if it weren't for people like you!!
 		</p>
-	</div>
\ No newline at end of file
+	</div>
```

Comment:
@SStrausman Is this missing a </div> at the end of  app/views/welcome/support_us.html.erb?